### DISCUSSION TITLE
wofi can't placed at the top level in full screen when updated to 0.27

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [hyprwm/Hyprland#2790] [General]
Does anyone know what is going on?please!

**Comment by MightyPlaza** [comment]
wofi is using top layer instead of overlay like rofi
wofi is unmaintained afaik so I wouldn't count on getting it fixed

**Reply by DreamMaoMao** [reply]
well,i will replace it by tofi

**Comment by DreamMaoMao** [comment]
> wofi is using top layer instead of overlay like rofiwofi 使用顶层而不是像 rofi 那样的覆盖 wofi is unmaintained afaik so I wouldn't count on getting it fixedwofi 是未维护的 afaik，所以我不会指望修复它
wofi also support ovlerlay,i add layer=overlay in config  , it work
